### PR TITLE
[1019] Create frontend URL helper

### PR DESCRIPTION
**Pull request by tvararu**
This can be imported and used inside Controllers to redirect users to Frontend for certain controller actions.

A call is added to OrganisationController, wrapper with a feature flag.

### CODE REVIEW

**Comment by tvararu** [comment]
[Organisation details links to C# Publish](https://trello.com/c/aQH97fdv/1019-organisation-details-links-to-c-publish)

**Comment by tvararu** [comment]
After this is merged, two environment variables need to be defined to enable this:

```bash
url__frontend="https://www2.dev.publish-teacher-training-courses.service.gov.uk"
FEATURE_FRONTEND_ORGANISATION_SHOW="true"
```

File: src/ui/Startup.cs [hunk 1]
```diff
@@ -263,6 +263,7 @@ public void ConfigureServices(IServiceCollection services)
             });
             services.AddScoped<SearchAndCompare.UI.Shared.Features.IFeatureFlags, SearchAndCompare.UI.Shared.Features.FeatureFlags>();
             services.AddSingleton<ISearchAndCompareUrlService>(x => new SearchAndCompareUrlService(Configuration.GetValue("SearchAndCompare:UiBaseUrl", "")));
+            services.AddSingleton<IFrontendUrlService>(x => new FrontendUrlService(Configuration));
```
Comment (defong) [inline src/ui/Startup.cs]: services.AddSingleton<IFrontendUrlService, FrontendUrlService>();

Don't think you need to new it up

File: src/ui/appsettings.Development.json [hunk 2]
```diff
@@ -17,5 +17,9 @@
     "site": "https://localhost:44364",
     "profile": "https://signin-test-pfl-as.azurewebsites.net"
   },
-  "API_URL": "https://bat-dev-manage-courses-api-app.azurewebsites.net/"
+  "ManageCourses": {
+    "FrontendBaseUrl": "https://localhost:1234"
```
Comment (defong) [inline src/ui/appsettings.Development.json]: its 3000 not 1234

**Comment by tvararu** [comment]
@defong good spots!

**Pull request merged**